src/ctia/stores/es/store.clj

```diff
@@ -40,6 +40,10 @@
              [verdict :as ve]]
             [schema.core :as s]))
 
+(defn delete-state-indexes [{:keys [conn index config]}]
+  (when conn
+    (es-index/delete! conn (str index "*"))))
```

Comment:
@saintx no by default if you didn't enforce wildcard you don't need to change anything, actually it works with ES vanilla config from docker.

I think you miss the real intent of the wildcard, a call is made for each store, however as indexes can be split, for instance the event index slicing I need the wildcard.